crates/sui-core/src/consensus_adapter.rs

```diff
@@ -90,14 +90,15 @@ impl ConsensusAdapterMetrics {
             sequencing_acknowledge_latency: register_histogram_vec_with_registry!(
                 "sequencing_acknowledge_latency",
                 "The latency for acknowledgement from sequencing engine. The overall sequencing latency is measured by the sequencing_certificate_latency metric",
-                &["retry"],
+                &["retry", "position"],
```

Comment:
The main reason why I put this here is in order to have a direct correlation to the latency as metric. So then we can plot histograms of latency, but still filter based on the `position`. You are right though @mwtian that plotting histograms etc on the position it self it would be challenging - let me introduce separate metric on this.

Reply:
If this is the case, it seems `sequencing_certificate_latency` is more appropriate? I'm not sure if there is any correlation between position and `sequencing_acknowledge_latency`. Also, position has non-trivial cardinality and it shouldn't be used with too many other labels ..